gratipay/billing/exchanges.py

```diff
@@ -179,6 +179,7 @@ def create_card_hold(db, participant, amount):
     # ===============
 
     cents, amount_str, charge_amount, fee = _prep_hit(amount)
+    amount = charge_amount - fee
```

Comment:
> How do we call record_an_exchange in the success case?

We don't. For successful card holds, we don't record an exchange at all. We create an exchange when we capture them, and [this same statement](https://github.com/gratipay/gratipay.com/blob/master/gratipay/billing/exchanges.py#L215) is present there.